v3/src/components/slider/slider-component.tsx

```diff
@@ -17,6 +17,8 @@ import { EditableSliderValue } from "./editable-slider-value"
 import './slider.scss'
 
 const kAxisMargin = 30
+const kComponentTitleHeight = 25
+const kSliderComponentHeight = 98
```

Comment:
There is slider-component-height in slider.scss. I know there's some way to make use of that constant here rather than having to redundantly define it.

Reply:
`slider.scss` no longer has that constant. I had to remove the height styling from the css to be able to change the slider height dynamically.